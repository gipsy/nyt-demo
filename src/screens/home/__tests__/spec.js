import React from 'react'
import { render, fireEvent } from '@tests/utils/render-with-app-context'
import Home from '@screens/home'

describe('<Home />', () => {
  test('should match snapshot', () => {
    const { container } = render(<Home />)
    expect(container).toMatchSnapshot()
  })

  // test('should change URL on submit button', () => {
  //   const { container } = render(<Home />)
  //   const submitButton = container.querySelector('.ant-input-search-button')
  //   const inputField = container.querySelector('.ant-input')
  //   fireEvent.change(inputField, {target: {value: 'singapore'}})
  //   fireEvent.click(submitButton)
  //   console.log(container)
  // })
})
