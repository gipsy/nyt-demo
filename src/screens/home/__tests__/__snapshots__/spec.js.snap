// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> should match snapshot 1`] = `
.c2 {
  display: block;
}

.c1 {
  margin: 0 auto;
  width: 50%;
}

.c0 {
  font-family: 'Noto Serif KR',sans-serif;
  color: #000;
  text-align: center;
  width: 100%;
}

@media (max-width:600px) {
  .c1 {
    width: 100%;
  }
}

<div>
  <h2
    class="c0"
  >
    What news are you looking for ?
  </h2>
  <div
    class="c1"
  >
    <span
      class="ant-input-search c2 ant-input-search-enter-button ant-input-search-large ant-input-affix-wrapper ant-input-affix-wrapper-lg"
    >
      <input
        class="ant-input ant-input-lg"
        placeholder="input search text"
        type="text"
      />
      <span
        class="ant-input-suffix"
      >
        <button
          class="ant-btn ant-input-search-button ant-btn-primary ant-btn-lg"
          type="button"
        >
          <span>
            Search
          </span>
        </button>
      </span>
    </span>
  </div>
</div>
`;
